test/djinterop/engine/v3/track_table_test.cpp

```diff
@@ -274,49 +274,49 @@ BOOST_DATA_TEST_CASE(
     DEFINE_SETTER_VALID_TEST_CASE(engine_column, min_schema)       \
     DEFINE_SETTER_INVALID_TEST_CASE(engine_column, min_schema)
 
-DEFINE_GETTER_SETTER_TEST_CASES(play_order, es::schema_3_0_0)
```

Comment:
The idea is that the version supplied here is the _earliest_ version when a given column is introduced to the table (hence the variable is called `min_schema`).  You don't need to wholesale update this (same goes for `performance_data_table_test.cpp`).

Example from V2 schemas: https://github.com/xsco/libdjinterop/blob/main/test/djinterop/engine/v2/track_table_test.cpp#L278-L327